src/sdk/tests/web3/test_factory.py

```diff
@@ -1,10 +1,29 @@
-from src.sdk.web3.factory import nft_contract, w3, account, Web3Wrapper
-from src.sdk.web3.chain import get_network_settings_by_name
-from src.sdk.exception import InvalidProvider, InvalidPrivateKey
 import pytest
 import web3
 import eth_account
 import hexbytes
+from web3 import Web3
+from src.sdk.exception import InvalidProvider, InvalidPrivateKey
+from src.sdk.web3.factory import (
+    w3,
+    nft_contract,
+    account,
+    chain,
+    _kovan,
+    _rinkeby,
+    Web3Wrapper,
+    ChainWrapper,
+)
+
+from src.sdk.constants import (
```

Comment:
How are these defined? Env vars?

Reply:
Exactly, eg.

`KOVAN_PROVIDER = os.getenv("KOVAN_PROVIDER", "")`
all this vars are set from env vars